pkg/apis/networkoperator/v1/networkconfig_types.go

```diff
@@ -53,11 +53,14 @@ type NetworkConfigSpec struct {
 	DefaultNetwork DefaultNetworkDefinition `json:"defaultNetwork"`
 
 	// Additional networks to make available to pods. If they are specified,
-	// pods can request them via annotations.
-	//
-	// Specifying any additionalNetworks will enable Multus across the cluster.
+	// pods can request them via annotations when multiple networks are enabled.
```

Comment:
Nitpick: this implies that you're allowed to specify `AdditionalNetworks` along with `DisableMultiNetwork: "true"`, and it just won't work at runtime. But in fact it will be rejected at configuration-parsing time.